### ISSUE TITLE
use circle menus for vote actions

### ISSUE THREAD

**Issue by bmorel** [Unvanquished/Unvanquished#1753]
Here is the current vote interface: 
![2022-02-15-175218_1366x768_scrot](https://user-images.githubusercontent.com/1316300/154109703-aaf4abe3-1ac2-4d23-b774-741cbb9dde24.png)

The problems I see with it are:

* it is in the main menu and thus slow
* it is drown in a rather long list of options
* it lacks some critical options

My suggestion is to use a circle menu instead, and add some of the missing options in it. I do not know which bind would spawn such menu, though.

As for the options:

* the 4 yes/no entries
* kick bots (would be nice to have add bots, too, but that belongs to another feature request)
* call draw / admit defeat
* kick player
* forbid building to a teammate (this one feels quite important to me)

This would be 9 options total, maybe more could be added, I do not know (that's why I opening this, to talk about it).

**Comment by necessarily-equal** [comment]
I don't think we should have yes/no in the same menu, to be honest. Otherwise, I kinda like the idea.
